TUnit.Assertions/AssertConditions/AssertionResult.cs

```diff
@@ -1,32 +1,38 @@
-﻿namespace TUnit.Assertions.AssertConditions;
+﻿using System.Runtime.CompilerServices;
+using System.Text;
+
+namespace TUnit.Assertions.AssertConditions;
 
 public class AssertionResult
 {
     public bool IsPassed { get; }
-    public string Message
-    {
-        get
-        {
-            return _message.Value;
-        }
-    }
-    private readonly Lazy<string> _message;
+    public string Message { get; }
 
-    private AssertionResult(bool isPassed, Func<string> messageGenerator)
+    private AssertionResult(bool isPassed, string messageGenerator)
```

Comment:
Recommend correcting this parameter name back to the original `message` name

Reply:
Done! 😄